src/internal/collection/transaction.go

```diff
@@ -220,6 +221,13 @@ func (s *stm) Put(key, val string, ttl int64, ptr uintptr) error {
 	return nil
 }
 
+func (s *stm) PutLease(key, val string, lease v3.LeaseID, ptr uintptr) error {
+	s.Lock()
+	defer s.Unlock()
+	s.wset[key] = stmPut{val, ttl, v3.OpPut(key, val, v3.WithLease(lease)), ptr}
```

Comment:
Does the linter complain about no fields when creating the `stmPut` ?

Reply:
No.